lib/Ast.ml

```diff
@@ -2066,11 +2066,11 @@ end = struct
     (* exponential without memoization *)
```

Comment:
Line 1920: What about `curry`? The compiler accepts `[@curry]` and `[@ocaml.curry]`. Also see Fmt_ast.ml:785. But I find no occurrences of `@curry` in the tree, so maybe this isn't necessary.

Reply:
That’s right – no uses of it, and we don’t advertise it in our user-facing documentation.  Also, it doesn’t have a keyword form.  I think it’s best to keep the AST-changing rewriting minimal, so I stuck with the things we advertise and that have keywords; we could even probably get away with omitting `%exclave`, as it’s new enough that it’s not used yet.